### DISCUSSION TITLE
decrease the body of SeisSol by moving pre-/post-process to another repo.

### DISCUSSION THREAD

**Discussion by daisy20170101** [SeisSol/SeisSol#1079] [Ideas]
I recently installed and compiled SeisSol on NeSI, NZ national HPC, and I released that how large SeisSol repo. is. It might be helpful to move something to another repo, in order to keep excutable files in the same but limited place, like $HOME.  Just an idea.

**Comment by sebwolf-de** [comment]
I just checked now. A fresh clone of SeisSol is 257MB large. Quite large, but nothing extraordinary. 149MB are the `.git` repository, which will stay, even if we move some things (pre/postprocessing have 23MB together) elsewhere. submodules add another14MB (6MB in `submodules` and additional 8MB in `.git`). The build folder adds another 48MB. In total, one SeisSol installation takes 320MB of space. Not perfect, but on HPC surely not a problem.

While there are good reasons to move pre-/postprocessing to another repo, this won't solve your issues with limited space on $HOME. 

```
❯ du -h -d 1 SeisSol
48M     SeisSol/build
6.3M    SeisSol/submodules
6.0M    SeisSol/src
88K     SeisSol/site_scons
1.8M    SeisSol/preprocessing
21M     SeisSol/postprocessing
36M     SeisSol/generated_code
456K    SeisSol/external
160K    SeisSol/cmake
232K    SeisSol/auto_tuning
44M     SeisSol/Documentation
16K     SeisSol/.github
60K     SeisSol/.ci
157M    SeisSol/.git
320M    SeisSol
```

**Reply by sebwolf-de** [reply]
Why don't you install SeisSol in $WORK? This should have plenty of space.

**Comment by daisy20170101** [comment]
Thanks for checking! I will move to $WORK. I also agree that there is other reasons to move them:-P
